package.json

```diff
@@ -108,5 +109,21 @@
   "files": [
     "bin",
     "dist"
-  ]
+  ],
+  "nyc": {
+    "all": true,
+    "include": [
+      "dist/htmlhint.js"
+    ],
+    "exclude": [
+      "**/*.spec.js",
+      "src/**/*.js",
+      "bin/**/*.js"
```

Comment:
`bin/**/*.js` will not exist in `beta` anymore
There is only `bin/htmlhint` (without any extension)
And this points to `dist/cli/htmlhint.js`

Should we move the target to `beta`?

Reply:
If we merge to dev, then we will change this in the beta branch.

Reply:
ok